index.html

```diff
@@ -8,7 +8,8 @@
     <link rel="stylesheet"
         href="https://rsms.me/inter/inter.css">
     <link rel="stylesheet" href="index.css">
-    <title>Random Password Generator</title>
+    <title>Random Password Generator - Can you help Frank?
```

Comment:
You probably don't want my name on your app son 🤭

Remember, once its in git you can't remove it

Reply:
Or not so easily at least..

Reply:
Ah ok. So to make another branch I had to make a change to a file so thought I'd just do it in the title for the minute.